do_everything.sh

```diff
@@ -433,6 +435,23 @@ echo
 
 (cd $prefix/nodelet_sample_app && ant debug)
 
+echo
+echo -e '\e[34mCreating image transport sample app.\e[39m'
+echo
+# Copy opencv 3rdparty libraries. These are needed to build the compressed image transport plugin.
+( cp $prefix/roscpp_android_ndk/share/OpenCV/3rdparty/lib/* $prefix/roscpp_android_ndk/lib/)
```

Comment:
I think this way is good for now but I would put this code in the `build_library.sh` file with a conditional check for "opencv".